main.py

```diff
@@ -1,10 +1,50 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main():
-    pass
+
+    with open("players.json") as f:
+        data = json.load(f)
+
+    for name in data:
```

Comment:
better use sctructure
name, information in data.values()
No need to use data[name], just information(or another name)